flaskr_tests.py

```diff
@@ -62,14 +81,58 @@ def test_edit(self):
             follow_redirects=True)
         self.app.get('/entry=1', follow_redirects=True)
         rv = self.app.post(
-            '/entry=1', 
-            data = dict(title='Bye',
-                        text='Nothing is allowed here'),
+            '/entry=1',
+            data=dict(title='Bye',
+                      text='Nothing is allowed here'),
             follow_redirects=True)
         assert b'HTML' not in rv.data
         assert b'Entry was successfuly edited' in rv.data
         assert b'Bye' in rv.data
 
+    def test_delete(self):
+        """Test entry delete."""
+        self.login('admin', 'default')
+        self.app.post(
+            'add',
+            data=dict(title='Hey',
+                      text='World'),
+            follow_redirects=True)
+        rv = self.app.post('delete=1', follow_redirects=True)
+        assert b'Hey' not in rv.data
+        assert b'The entry was deleted' in rv.data
```

Comment:
I think I understand it now, you have the HTML in there, so it's hard to check strictly.